app/controllers/mock.php

```diff
@@ -646,6 +649,69 @@
         $response->noContent();
     });
 
+App::get('/v1/mock/github/callback')
+    ->desc('Create installation document using GitHub installation id')
+    ->groups(['mock', 'api', 'vcs'])
+    ->label('scope', 'public')
```

Comment:
Let's add `->label('docs', false)`, I think it prevents this method to getting to SDKs